products/centos-stream.md

```diff
@@ -0,0 +1,41 @@
+---
+title: CentOS Stream
+category: os
+iconSlug: centos-stream
+permalink: /centos-stream
+versionCommand: lsb_release --release
+releasePolicyLink: https://wiki.centos.org/About/Product
+releaseLabel: "CentOS Stream __RELEASE_CYCLE__"
```

Comment:
Now that CentOS and CentOS Stream are not mixed on the same page this does not seem necessary anymore.